### PR TITLE
128 question event display

### PR DESCRIPTION
**Pull request by ivarhe**

### CODE REVIEW

File: web/app/components/Question.tsx [hunk 1]
```diff
@@ -1,30 +1,40 @@
+import { ActionTypes } from '~/context/game/game'
 import { useGameContext } from '~/context/game/GameContext'
-import { IAlternative, ScoreboardProps } from '~/context/QuizContext'
+import { IAlternative } from '~/context/QuizContext'
+import { useWebSocket } from '~/context/SocketContext'
 import AnswerButton from './AnswerButton'
 
 
-export function Question(tb: ScoreboardProps): JSX.Element {
+export function Question(): JSX.Element {
```
Comment (perkynades) [inline web/app/components/Question.tsx:8]: Update to return ReactElement instead

File: web/app/context/QuizContext.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { ReactElement, useState } from 'react'
 import { useContext } from 'react'
 
 export interface IAlternative {
+    id: number
```
Comment (perkynades) [inline web/app/context/QuizContext.tsx:5]: Make this optional, or else it will break quiz creation

**Review by perkynades — CHANGES_REQUESTED** [review]

**Review by Henrhh — APPROVED** [review]
lgtm

File: web/app/components/Question.tsx (hunk 1 above)
Comment (ivarhe) [reply web/app/components/Question.tsx]: 🙌

**Review by perkynades — APPROVED** [review]
GLHF

**Pull request merged**